.gitignore

```diff
@@ -18,4 +18,4 @@ src/device/kendryte/*.go
 src/device/kendryte/*.s
 vendor
 llvm-build
-llvm-project
+llvm-project
```

Comment:
This is an unrelated change that should not be included in this PR (although it is a good change).